Firmware/mmu2_reporting.cpp

```diff
@@ -100,7 +100,14 @@ extern void ReportErrorHookSensorLineRender()
 static uint8_t ReportErrorHookMonitor(uint8_t ei) {
     uint8_t ret = 0;
     bool two_choices = false;
-    static int8_t enc_dif = 0;
+    static int8_t enc_dif = lcd_encoder_diff;
```

Comment:
Are you expecting this to repeatedly initialize enc_diff to the same value as lcd_encoder_diff? If so it won't do that, static variables are only initialized once the first time they are used.

Reply:
Yes the static is the intended behaviour. 👍 The function is called very often to be non-blocking. We need to be able to track the rotation outside the monitor function.